### PR TITLE
update ingress to use www, remove header and add staging / prod

### PR DESCRIPTION
**Pull request by pixie79**

### CODE REVIEW

File: app/core/components/Header.jsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ class Header extends React.Component {
                     <div className="govuk-header__logo">
                         <a href="#" className="govuk-header__link govuk-header__link--homepage"
                            style={{textDecoration: 'none', pointerEvents: 'none'}}>
+/*
         <span className="govuk-header__logotype">
```
Comment (aminmc) [inline app/core/components/Header.jsx]: no need to comment

File: .drone.yml [hunk 2]
```diff
@@ -100,136 +165,70 @@ pipeline:
...
       - kd --insecure-skip-tls-verify -f kube/deployment.yml --timeout=5m
       - kd --insecure-skip-tls-verify -f kube/ingress.yml
     when:
       branch:
         - master
-      environment:
-        - demo
       event:
-        - deployment
+        - push
```
Comment (tomfitzherbert) [inline .drone.yml]: This should probably be a deployment event and not automatic.

File: .drone.yml [hunk 3]
```diff
@@ -26,6 +25,72 @@ pipeline:
       - ${DRONE_BUILD_NUMBER}
       - ${DRONE_COMMIT_SHA}
       - latest
+    when:
+      branch: master
+      event: push
+
+  deploy_to_dev:
```
Comment (tomfitzherbert) [inline .drone.yml]: We seem to have `deploy_to_dev` twice now.

File: .drone.yml [hunk 4]
```diff
@@ -100,136 +165,70 @@ pipeline:
...
-#      - kd --insecure-skip-tls-verify -f kube/deployment.yml --timeout=5m
-#      - kd --insecure-skip-tls-verify -f kube/ingress.yml
-#    when:
-#      branch:
-#        - master
-#      event:
-#        - push
-
-  deploy_to_demo:
+  deploy_to_production:
```
Comment (tomfitzherbert) [inline .drone.yml]: All the prod env vars are pointing at dev versions

**Pull request merged**